### PR TITLE
New information architecture

### PR DESCRIPTION
**Pull request by charypar**
This pulls in the new IA from next-navigation, and also make a couple other visual tweaks to the navigation UI itself.

**requires next-express 14.2.2 or later**

/cc @ironsidevsquincy @i-like-robots

### CODE REVIEW

File: src/scss/header-v2/_navigation.scss [hunk 1]
```diff
@@ -33,6 +27,12 @@ label > * {
 	height: 100%;
 	background: oColorsGetPaletteColor('cold-2');
 }
+.next-navigation-v2__group__items {
+	@include oGridRespondTo($until: M) {
+		background: center url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAyAAAAAFCAYAAAC9+YhwAAAAAXNSR0IArs4c6QAAAIFJREFUaAXt16ENACAMBEBgpU6MZgG2gwVogqi8yk/z4tz3iDjNESBAgMCXwF7z688TAQIECBAg8BYY71hKgAABAgQIECBAgACBegEDpN5UIwECBAgQIECAAAECiYABksCICRAgQIAAAQIECBCoFzBA6k01EiBAgAABAgQIECCQCFyl4wOmY4gKVQAAAABJRU5ErkJggg==');
+		background-position: 42%;
```
Comment (i-like-robots) [inline src/scss/header-v2/_navigation.scss]: Magic number klaxon! Anything we can do here to make this implicit?

File: src/scss/header-v2/_navigation.scss [hunk 2]
```diff
@@ -118,11 +117,19 @@ label > * {
...
 		text-align: right;
 		padding: 0 2px 0 0;
-		&:after { @include dark_border_bottom; }
+
+		span:after {
+			@include nextIcon(arrow-right, oColorsGetColorFor(nav, text), 10);
+			content: '';
+			padding-left: 1.3em;
+			margin-bottom: -0.05em;
+			margin-right: -1em;
```
Comment (i-like-robots) [inline src/scss/header-v2/_navigation.scss]: Font sizes, padding etc. are all defined in px - is switching to ems here helpful?

File: src/scss/header-v2/_navigation.scss [hunk 3]
```diff
@@ -187,7 +190,7 @@ label > * {
 	padding: 0 0 20px;
 	position: absolute;
 	left: 0;
-	transition: 0.2s opacity 0.25s ease, 0s left 0.25s step-end;
+	transition: 0.1s opacity 0.25s ease, 0s left 0.25s step-end;
```
Comment (i-like-robots) [inline src/scss/header-v2/_navigation.scss]: Do we _really_ want the delay?

File: src/scss/header/_base.scss [hunk 4]
```diff
@@ -25,6 +25,14 @@
 			}
 		}
 	}
+
+	a {
+		border-bottom: 0;
+	}
+
+	a:hover {
+		color: inherit;
```
Comment (i-like-robots) [inline src/scss/header/_base.scss]: Much easier :+1:

Comment (kaelig) [inline src/scss/header/_base.scss]: What about the focus state?

Comment (charypar) [inline src/scss/header/_base.scss]: @kaelig You can't stay away ;) And you're right, missed that one

File: src/scss/header-v2/_navigation.scss (hunk 3 above)
Comment (charypar) [inline src/scss/header-v2/_navigation.scss]: We do, strangely, the animation basically doesn't happen without it... not sure if it's performance related or what, but it doesn't fade in any more without the delay :(

Needs more investigation

File: src/scss/header-v2/_navigation.scss (hunk 2 above)
Comment (charypar) [inline src/scss/header-v2/_navigation.scss]: The positioning of the chevron is only related to the font metrics, bigger fonts will need more space, so changing the font-size should automatically affect these. That's my thinking behind it.

File: src/scss/header-v2/_navigation.scss (hunk 1 above)
Comment (charypar) [inline src/scss/header-v2/_navigation.scss]: I know!! It's terrible... the image itself is asymmetric, but it's not quite enough. The number is related to the number of columns - it is basically 5/12. Any ideas?

File: src/scss/header-v2/_navigation.scss [hunk 5]
```diff
@@ -16,12 +20,6 @@ label > * {
 
 .next-header-v2 .next-header__row--primary {
 	&:before,
```
Comment (i-like-robots) [inline src/scss/header-v2/_navigation.scss]: redundant selector

File: src/scss/header-v2/_navigation.scss (hunk 3 above)
Comment (i-like-robots) [inline src/scss/header-v2/_navigation.scss]: Probably worth noting the workaround in the code as a `// HACK:`

File: src/scss/header/_masthead.scss [hunk 6]
```diff
@@ -37,6 +37,10 @@
 			width: 336px;
 		}
 
+		&:hover {
+			color: transparent;
```
Comment (i-like-robots) [inline src/scss/header/_masthead.scss]: Better to put the text in a `span.n-utils-visually-hidden`?

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**